MonstieWash/Assets/Scripts/PlayerHand.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Playerhand : MonoBehaviour
+{
+    //Unity Inspector
+	[SerializeField]
+    [Range(1.0f, 50.0f)]
+	private float cursorSpeed = 20;
+
+    //Private
+    private Transform m_Transform;
+    private Vector3 handPosition;
+	private float moveHorizontal;
+	private float moveVertical;
```

Comment:
I'm not sure if we've finalized this yet so don't worry about it too much for now, but it'd be good to be using the notation mentioned in the code guide on notion for all member variables (to specify that they are a member of this class). The `m_transform` is what you want to be doing for all member variables for now. (Like below)

```suggestion
    private Transform m_transform;
    private Vector3 m_handPosition;
	private float m_moveHorizontal;
	private float m_moveVertical;
```